src/start.js

```diff
@@ -79,16 +80,15 @@ let enableDebugger = process.argv.find(arg => arg.includes('enableDebugger'))
 const DEBUG_MODE = !!process.env.STETHOSCOPE_DEBUG
 
 const focusOrCreateWindow = (mainWindow) => {
-  if (mainWindow && !mainWindow.isDestroyed()) {
+  if (mainWindow) {
     if (mainWindow.isMinimized()) {
-      mainWindow.restore()
+      return mainWindow.restore()
     }
```

Comment:
Can we combine these?
`mainWindow && mainWindow.isMinimized()`

Reply:
We could, but there are two handled conditions based on `mainWindow` not being `null`, if it's minimized we restore, otherwise we destroy the window instance and rebuild. Happy to restructure if you feel it would be cleaner